### ISSUE TITLE
order of das tool env

### ISSUE THREAD

**Issue by taylorreiter** [metagenome-atlas/atlas#223]
In atlas/atlas/envs/DASTool.yaml, channels are currently listed as:
```
channels:
    - bioconda
    - r
    - ruby-lang
    - conda-forge
    - defaults
dependencies:
    - das_tool=1.1.1
```

i think the channel order should have conda-forge so it has priority over bioconda. 

```
channels:
    - conda-forge
    - bioconda
    - r
    - ruby-lang
    - defaults
dependencies:
    - das_tool=1.1.1
```

could this fix #222 ?

**Comment by taylorreiter** [comment]
I ran this locally and can confirm that this fixes #222 for me.

**Comment by SilasK** [comment]
Great, thank you for testing this out. Do you want to make a Pull request?

**Comment by taylorreiter** [comment]
that would be awesome, yes!

**Comment by taylorreiter** [comment]
Do you think the bioconda recipe could be updated soon as well? I am hoping to teach atlas at a 60 person workshop tomorrow morning... :) I can walk everyone through how to change the hiding yaml file, but I would really rather not. I also know you'll have little control over the merge speed of the bioconda folk

**Comment by SilasK** [comment]
Ah great @taylorreiter, I'm not sure If followed all issues. 
 It's only this, change the das tool recipe?

**Comment by taylorreiter** [comment]
so this will fix #223 (this issue) and #222. 

#224 is still unsolved, and my tinkering has not yet found a solution to fix it.
